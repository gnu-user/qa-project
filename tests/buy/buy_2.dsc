TITLE
    Buy Transaction Privileges (sell-standard)
DESCRIPTION
    Run the login transaction using a buy-standard account. Initiate the buy 
    transaction. Verify that the transaction is rejected and an error occurs.
DEPENDENCIES
