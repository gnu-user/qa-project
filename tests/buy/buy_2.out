Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Invalid transaction, you do not have the necessary privileges.